crates/core/src/kvs/mem/mod.rs

```diff
@@ -130,7 +136,17 @@ impl super::api::Transaction for Transaction {
 		}
 		// Mark the transaction as done.
 		self.done = true;
-		// Commit the transaction.
+		/*
+		// Execute on the blocking threadpool
+		affinitypool::spawn_local(|| -> Result<_, Error> {
+			// Commit this transaction
+			self.inner.commit();
+			// Continue
+			Ok(())
+		})
+		.await?;
+		*/
```

Comment:
Unused?
```suggestion
		/*
		// Execute on the blocking threadpool
		affinitypool::spawn_local(|| -> Result<_, Error> {
			// Commit this transaction
			self.inner.commit();
			// Continue
			Ok(())
		})
		.await?;
		*/
```

Reply:
This will go in as soon as the SurrealKV new release is merged in (after `v2.2.0`), so keeping it here for that reason 👍

Reply:
The async commit in SurrealKV is changing to synchronous commit in the next release, in this PR: https://github.com/surrealdb/surrealdb/pull/5427